biome.jsonc

```diff
@@ -24,7 +24,10 @@
   },
   "formatter": {
     "enabled": true,
-    "formatWithErrors": false,
+    "formatWithErrors": true,
+    "indentStyle": "space",
+    "indentWidth": 2,
+    "lineEnding": "lf",
```

Comment:
This should be taken from the .editorconfig file. If not, then we should set l36 to false and delete .editorconfig again.